.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Build and push docker images
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  build_core:
+    name: Build image
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Build and push the image
+        uses: docker/build-push-action@v1.1.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          dockerfile: ./Dockerfile
+          repository: certpl/karton-system
```

Comment:
Maybe
```
           repository: certpl/${{ github.event.repository.name }} 
```
for consistency with other kartoniks?

Reply:
But we want to ship karton-system instead of "karton"